compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +166,63 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init(true);
+
+  // check if Custom(AddV2) exists before the pass
+  auto addV2 = dynamic_cast<luci::CircleCustom *>(_g.addV2());
+  EXPECT_NE(nullptr, addV2);
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  // check if Custom(AddV2) is converted to Add
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+
+  // check if Custom(BroadcastTo) is removed
+  auto input_y = dynamic_cast<luci::CircleConst *>(add->y());
+  EXPECT_NE(nullptr, input_y);
+}
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_BuiltinBroadcastTo)
+{
+  _g.init(false);
+
+  // check if Custom(AddV2) exists before the pass
+  auto addV2 = dynamic_cast<luci::CircleCustom *>(_g.addV2());
+  EXPECT_NE(nullptr, addV2);
```

Comment:
```suggestion
  auto addV2 = dynamic_cast<luci::CircleCustom *>(_g.output(0)->from());
  EXPECT_NE(nullptr, addV2);
  EXPECT_TRUE("AddV2" == addV2->custom_code());
```

Reply:
I modified the code appropriately by reflecting your comment as follows. :)
```
// check if Custom(AddV2) exists before the pass
  auto addV2_out = dynamic_cast<luci::CircleCustomOut *>(_g.output(0)->from());
  auto addV2 = dynamic_cast<luci::CircleCustom *>(addV2_out->input());
  EXPECT_NE(nullptr, addV2);
  EXPECT_TRUE("AddV2" == addV2->custom_code());
```

Reply:
```suggestion
   // check if Custom(AddV2) exists before the pass
  auto addV2_out = dynamic_cast<luci::CircleCustomOut *>(_g.output(0)->from());
  auto addV2 = dynamic_cast<luci::CircleCustom *>(addV2_out->input());
  EXPECT_NE(nullptr, addV2);
  EXPECT_TRUE("AddV2" == addV2->custom_code());
```